packages/sdks/output/angular/scripts/multi-build.mjs

```diff
@@ -0,0 +1,103 @@
+import babel from '@babel/core';
+import tsPlugin from '@babel/plugin-syntax-typescript';
+import { execSync } from 'child_process';
+import fs from 'fs';
+import glob from 'glob';
+import path from 'path';
+import prettier from 'prettier';
+
+/**
+ * @typedef {import('@babel/core').PluginObj} babel.PluginObj
+ * @typedef {babel.PluginObj} InlinePlugin
+ */
+
+const sdkEnv = process.env.SDK_ENV;
+
+if (!sdkEnv) {
+  throw new Error('SDK_ENV is required to build the SDK.');
+}
+
+const ngPackageJsonPath = path.resolve('ng-package.json');
+const ngPackageJson = JSON.parse(fs.readFileSync(ngPackageJsonPath, 'utf-8'));
+
+const outputPath = {
+  node: './lib/node',
+  browser: './lib/browser',
+}[sdkEnv];
+
+if (!outputPath) {
+  throw new Error(
+    `Unknown SDK_ENV: ${sdkEnv}. Expected one of 'node', 'browser'.`
+  );
+}
+
+// Set the destination path to the correct output path for specific SDK_ENV
+ngPackageJson.dest = outputPath;
+
+fs.writeFileSync(
+  ngPackageJsonPath,
+  // we needed to prettify the JSON to make sure it matches the linting rules
+  await prettier.format(JSON.stringify(ngPackageJson, null, 2), {
+    parser: 'json',
+  }),
+  'utf-8'
+);
+
+const foundFiles = glob.sync('src/**/*/choose-eval.*');
+
+if (foundFiles.length !== 1) {
+  throw new Error(
+    'Expected exactly one file named `choose-eval.ts` containing "placeholder-runtime" import.'
+  );
+}
```

Comment:
you should add a big loud comment at the top of `choose-eval.ts` saying something like:

```
/*
 * THIS FILE IS EXPECTED TO EXIST WITH THIS EXACT NAME BY SUBSEQUENT BUNDLE STEPS THAT TRANSFORM IT.
 * DO NOT RENAME THIS FILE.
 */
```

Reply:
done